Add Docker Desktop for Apple Silicon RC 2 (#12577)
docker/docker.github.io: docker-for-mac/apple-m1.md

>
> We encourage you to try the release candidate and report any issues in the [Docker Desktop for Mac GitHub](https://github.com/docker/for-mac) repository.

## Docker Desktop RC 2

2021-03-25

> Click on the following link to download the latest release candidate of Docker Desktop for Apple Silicon.

Reviewer: Insert:

This build allows users to switch between qemu-based and hypervisor.framework-based virtual machines. The default is qemu, but it can be switched back to hypervisor.framework in the experimental tab of the Preferences.

This is still a tech preview, and we are not committing to include either of these specific back ends for the final release. Which we release eventually will depend on user feedback. We hope that the new qemu back end will avoid some of the known issues listed below, and we would welcome confirmation of this. However, it may introduce other issues that we have not discovered in in-house testing, so we would also welcome reports of problems with it, especially areas where it is inferior to the hypervisor.framework.

> > [Download](https://desktop.docker.com/mac/stable/arm64/62293/Docker.dmg)

Author: New build number should be: 62345

> ### Known issues

Reviewer: Delete "we are working with our partners on these".

Then above the VPN bullet, add new rubric:

The following issues are seen when using the hypervisor.framework back end.

Author: I think "hypervisor.framework" should be "virtualization.framework" here (qemu uses the "hypervisor.framework" and avoids the issues)

>
> We encourage you to try the release candidate and report any issues in the [Docker Desktop for Mac GitHub](https://github.com/docker/for-mac) repository.

## Docker Desktop RC 2

> 2021-03-25

Reviewer 2: ```suggestion
2021-03-26
```

Click on the following link to download the latest release candidate of Docker Desktop for Apple Silicon.

> [Download](https://desktop.docker.com/mac/stable/arm64/62345/Docker.dmg)

> This build allows users to switch between `qemu`-based and `virtualization.framework`-based virtual machines. The default is `qemu`, but it can be switched back to `virtualization.framework` in the experimental tab of the Preferences.

Reviewer 2: ```suggestion
This build allows you to switch between `qemu`-based and `virtualization.framework`-based virtual machines. The default is `qemu`, however, you can switch it back to `virtualization.framework` using the **Preferences** > **Experimental** tab.
```

> This build allows you to switch between `qemu`-based and `virtualization.framework`-based virtual machines. The default is `qemu`, however, you can switch it back to `virtualization.framework` using the **Preferences** > **Experimental** tab.

Reviewer 3: ```suggestion
With this build, we have now defaulted to `qemu`-based virtual machine which we believe resolves some of the issues noted as known issues in the previous release candidate. You will also be able to switch between `qemu`-based and `virtualization.framework`-based virtual machines using the **Preferences** > **Experimental** tab.
```

> With this build, we have now defaulted to `qemu`-based virtual machine which we believe resolves some of the issues noted as known issues in the previous release candidate. You will also be able to switch between `qemu`-based and `virtualization.framework`-based virtual machines using the **Preferences** > **Experimental** tab.

Reviewer: ```suggestion
In this build, we have defaulted to a `qemu`-based virtual machine, which we believe resolves some of the issues noted as known issues in the previous release candidate. You can switch between `qemu`-based and `virtualization.framework`-based virtual machines using the **Preferences** > **Experimental** tab.
```

> This is still a tech preview, and we are not committing to include either of these specific back ends for the final release. Which we release eventually will depend on user feedback. We hope that the new `qemu` back end will avoid some of the known issues listed below, and we would welcome confirmation of this. However, it may introduce other issues that we have not discovered in in-house testing, so we would also welcome reports of problems with it, especially areas where it is inferior to the `virtualization.framework`.

Reviewer 2: ```suggestion
Apple Silicon for Docker Desktop is still a tech preview. We are not committing to include either of these specific backends for the final release. The backend that we choose to release eventually depends on user feedback. We hope that the new `qemu` backend will avoid some of the known issues listed below. We welcome your confirmation on this. However, it may introduce other issues that we have not discovered in in-house testing, so we would also welcome any reports of problems with it, especially areas where it is inferior to the `virtualization.framework`.
```

> Docker Desktop for Apple Silicon is still a tech preview. We are not committing to include either of these specific backends for the final release. The backend that we choose to release eventually depends on user feedback. We hope that the new `qemu` backend will avoid some of the known issues listed below. We welcome your confirmation on this. However, it may introduce other issues that we have not discovered in in-house testing, so we would also welcome any reports of problems with it, especially areas where it is inferior to the `virtualization.framework`.

Reviewer 3: ```suggestion
The backend that we choose for the GA release  depends on user feedback and bug reports. We are interested in your feedback in whether the known issues noted below are resolved with the `qemu` backend. We are equally interested in whether the `qemu` backend introduces other issues that we have not discovered in our in-house testing. Please help us by reporting any issues you have with this build, particularly in areas where your experience with the `qemu` backend performs worse than the `virtualization.framework` backend.
```

> The backend that we choose for the GA release  depends on user feedback and bug reports. We are interested in your feedback in whether the known issues noted below are resolved with the `qemu` backend. We are equally interested in whether the `qemu` backend introduces other issues that we have not discovered in our in-house testing. Please help us by reporting any issues you have with this build, particularly in areas where your experience with the `qemu` backend performs worse than the `virtualization.framework` backend.

Reviewer: ```suggestion
The backend that we choose for the GA release will depend on user feedback and bug reports. We are interested in your feedback on whether the known issues noted below are resolved with the `qemu` backend. We are equally interested in whether the `qemu` backend introduces other issues that we have not discovered in our in-house testing. Please help us by reporting any issues you have with this build, particularly in areas where your experience with the `qemu` backend is worse than the `virtualization.framework` backend.
```

   We are still gathering data and testing alternate kernel versions.

### Fixes since Docker Desktop RC 1

> - Inter-container HTTP and HTTPS traffic is now routed correctly. Previously it was accidentally forwarded to an HTTP proxy on the host. Fixes [docker/for-mac#5476](https://github.com/docker/for-mac/issues/5476).

Reviewer: Omit middle sentence. (Focus on the customer symptom, not the cause of the bug).
